.snippets/text/common/general-py-tutorial-check.md

```diff
@@ -0,0 +1,5 @@
+
+!!! note
+    The examples in this guide are based on a MacOS or Ubuntu 20.04 environment. If you're using Windows, you'll need to adapt them accordingly.
+    
+    Furthermore, please ensure that you have Python3 and a package manager like PIP installed.
```

Comment:
I don't think PIP is ever capitalized like this 🤔 probably not a big deal, but it just looked a little off